src/test/java/MainTest.java

```diff
@@ -1,23 +1,17 @@
-import no.nav.apiapp.ApiApp;
-import no.nav.fo.veilarbvedtakinfo.TestContext;
-import no.nav.fo.veilarbvedtakinfo.config.ApplicationTestConfig;
-import no.nav.fo.veilarbvedtakinfo.db.DbTestUtils;
-import no.nav.testconfig.ApiAppTest;
-
-import static java.lang.System.getProperty;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import static no.nav.common.utils.EnvironmentUtils.NAIS_APP_NAME_PROPERTY_NAME;
 import static no.nav.fo.veilarbvedtakinfo.config.ApplicationConfig.APPLICATION_NAME;
-import static no.nav.testconfig.ApiAppTest.Config.builder;
 
 public class MainTest {
+
     private static final String PORT = "8800";
 
     public static void main(String[] args) {
+
         String[] arguments = {PORT};
 
-        ApiAppTest.setupTestContext(builder().applicationName(APPLICATION_NAME).build());
-        TestContext.setup();
-        DbTestUtils.setupContext(getProperty("database"));
+        System.setProperty(NAIS_APP_NAME_PROPERTY_NAME, APPLICATION_NAME);
```

Comment:
Bruk heller standard måten og starte opp spring boot lokalt på. Sjekk i en annen spring boot app for hvordan det kan gjøres